### PR TITLE
ign -> gz Header Migration : ign-math

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/ignition-tooling/release-tools/issues/711

### CODE REVIEW

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Comment by methylDragon** [comment]
Extra files have been deleted.

I'm not sure if I edited the Bazel buildfile correctly.

File: include/ignition/math/MassMatrix3.ipynb [hunk 1]
```diff
@@ -0,0 +1,134 @@
+{
```
Comment (scpeters) [inline include/ignition/math/MassMatrix3.ipynb:1]: this file should be deleted

**Review by scpeters — COMMENTED** [review]
I believe this also has some of the same issues as ign-utils:

* remove `ignition/math/math.hh` redirection header
* add `ignition/math/detail/Export.hh` redirection header

File: include/ignition/math/MassMatrix3.ipynb (hunk 1 above)
Comment (methylDragon) [reply include/ignition/math/MassMatrix3.ipynb:1]: Noting: git picked up the math.hh deletion and Export creation as a move for some reason.

**Comment by methylDragon** [comment]
Ubuntu CI is throwing a warning (Python not found), I don't think this is related to this PR.

**Comment by methylDragon** [comment]
We just need https://github.com/ignitionrobotics/ign-math/pull/414 merged for CI to pass, I think.

File: include/ignition/math/eigen3.hh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/*
```
Comment (scpeters) [inline include/ignition/math/eigen3.hh:1]: I would have expected this file in the `eigen/include/ignition/math` folder

Comment (methylDragon) [reply include/ignition/math/eigen3.hh]: Updated! https://github.com/ignitionrobotics/ign-math/pull/413/commits/aca4520863e633ea934a796e51a056c7606510d0

**Review by scpeters — APPROVED** [review]

**Comment by scpeters** [comment]
I think we missed the headers from https://github.com/ignitionrobotics/ign-math/pull/412 when rebasing

**Pull request merged**